src/js/base/stories/NoneFoundStories/NoneFoundSection.stories.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { NoneFoundSection } from "../../NoneFound";
+
+export default {
+    title: "base/NoneFound/NoneFoundSection",
+    component: NoneFoundSection,
+    args: {
+        noun: "testNoun"
```

Comment:
```suggestion
        noun: "items"
```
If you want to add a more general case to show where things get substituted I would do that in a story